packages/cloudi/cloudi.1.7.2/opam

```diff
@@ -0,0 +1,17 @@
+opam-version: "1.2"
+name: "cloudi"
+version: "1.7.2"
+maintainer: "Michael Truog <mjtruog@gmail.com>"
+authors: "Michael Truog <mjtruog@gmail.com>"
+homepage: "http://cloudi.org/"
+bug-reports: "https://github.com/CloudI/CloudI/issues"
+license: "MIT"
+dev-repo: "https://github.com/CloudI/cloudi_api_ocaml.git"
+build: [
+  [make]
+]
+remove: ["ocamlfind" "remove" "cloudi"]
```

Comment:
as you are using a `.install` file to install your package, you don't need that `remove` field: this will be taken care automatically by opam